_layouts/landing_page.html

```diff
@@ -12,7 +12,50 @@
     <body>
 {% include header.html %}
         <main id="landing-page-main" role="main" class="content-wrapper">
+            <div style="text-align: center;">
+                <img src="/assets/grid_hero_bg.jpg" alt="Hyperledger Grid Banner">
+            </div>
             <div id="landing-page-top-section" class="landing-page-section">
+                <p>
+                    Hyperledger Grid is a platform for building supply chain
+                    solutions that include distributed ledger components. This
```

Comment:
maybe "solutions that include distributed ledger capabilities" given we used components again in next line.